backend/coins/btc/handlers/handlers.go

```diff
@@ -381,15 +391,37 @@ func (handlers *Handlers) getUTXOs(*http.Request) (interface{}, error) {
 }
 
 func (handlers *Handlers) getAccountBalance(*http.Request) (interface{}, error) {
-	balance, err := handlers.account.Balance()
+	type balance struct {
+		HasAvailable bool            `json:"hasAvailable"`
+		Available    FormattedAmount `json:"available"`
+		HasIncoming  bool            `json:"hasIncoming"`
+		Incoming     FormattedAmount `json:"incoming"`
+	}
+
+	type result struct {
+		Success   bool    `json:"success"`
+		ErrorCode string  `json:"errorCode,omitempty"`
+		Balance   balance `json:"balance,omitempty"`
+	}
+	accountBalance, err := handlers.account.Balance()
 	if err != nil {
-		return nil, err
+		ret := result{
+			Success: false,
+		}
+		if errp.Cause(err) == accounts.ErrSyncInProgress {
```

Comment:
Could do a me general check:

```
		if errCode, ok := errp.Cause(err).(errp.ErrorCode); ok {
			...
		}
```

(I found this in backend/handlers/handlers.go)

Reply:
Removed the errocode in the api as discussed